### PR TITLE
Re: PR #28 - which I accidentally pushed to master instead of gh-pages.

### PR DESCRIPTION
**Pull request by ttimbers**
This commit contains new multiple choice questions tested at a recent SWC workshop (Feb 2015) for 02-create, 03-pipefilter, 04-loop, 05-script and 06-find. This commit also has some suggested changes for the wording of the setup for the first example of why you would want to use loops (creatures directory example). And finally, this commit also has a suggested change to correct the reason for the error output when you try mv to rename files using wild-cards instead of a loop.

### CODE REVIEW

File: 04-loop.md [hunk 1]
```diff
@@ -39,9 +39,10 @@ This wouldn't back up our files, instead we get an error
 mv: target `original-*.dat' is not a directory
 ~~~
 
-This is because there are no files matching the wildcard `original-*.dat`.
-In this case,
-Bash passes the unexpanded wildcard as a parameter to the `mv` command.
+This is a problem arises when mv receives more than two inputs. When this happens, it
```
Comment (r-gaia-cs) [inline 04-loop.md]: Wrap "mv" with backtick quotes:

~~~
This is a problem arises when `mv` receives more than two inputs. When this happens, it
~~~

File: 06-find.md [hunk 2]
```diff
@@ -393,6 +393,34 @@ North Whitehead wrote in 1911, "Civilization advances by extending the
...
+> With searching comes loss
+> and the presence of absence:
+> "My Thesis" not found.
+>
+> Yesterday it worked
+> Today it is not working
+> Software is like that.
+> ~~~
+>
+> From the above text, contained in the file haiku.txt, which command would result in the
```
Comment (r-gaia-cs) [inline 06-find.md]: Wrap "haiku.txt" with backtick quotes:

~~~
From the above text, contained in the file `haiku.txt`, which command would result in the
~~~

**Comment by r-gaia-cs** [comment]
@ttimbers Awesome work. :sparkles:

Could you make the two changes that I suggested? After you change the files you can

~~~
$ git add -u
$ git commit --amend
$ git push -f origin gh-pages
~~~

to update this pull request. After you update this pull request, please leave a note so we can merge this change. =)

File: 04-loop.md [hunk 3]
```diff
@@ -39,9 +39,10 @@ This wouldn't back up our files, instead we get an error
 mv: target `original-*.dat' is not a directory
 ~~~
 
-This is because there are no files matching the wildcard `original-*.dat`.
-In this case,
-Bash passes the unexpanded wildcard as a parameter to the `mv` command.
+This is a problem arises when `mv` receives more than two inputs. When this happens, it
```
Comment (ChristinaLK) [inline 04-loop.md]: last little comment - "This problem" not "This is a problem"

Comment (ChristinaLK) [inline 04-loop.md]: I'll fix this via github.

**Pull request merged**